index.js

```diff
@@ -170,15 +170,13 @@ function create (opts) {
       if (err) return cb(err) // no key found
 
       key = datKeyAs.str(key)
-      var feed = core.createFeed(key, {
-        storage: storage(path.join(dir, 'data', key.slice(0, 2), key.slice(2) + '.data'))
-      })
+      var feed = hypercore(storage(path.join(dir, 'data', key.slice(0, 2), key.slice(2) + '.data')), key)
+
       noContent.get(discKey, function (err) {
         if (!err) return done(null)
         feed.get(0, {wait: false}, function (err, data) {
           if (err) {
-            if (err.notFound) return done(null)
-            return cb(err)
+            if (!err.notFound) return cb(err)
```

Comment:
not sure how that got in there, I think I was experimenting on another branch. removing it